MediaBrowser.Model/Dto/ImageInfo.cs

```diff
@@ -30,6 +30,12 @@ public class ImageInfo
         /// <value>The path.</value>
         public string Path { get; set; }
 
+        /// <summary>
+        /// Gets or sets the blurhash.
+        /// </summary>
+        /// <value>The blurhash.</value>
+        public string Hash { get; set; }
```

Comment:
Should this be renamed to `BlurHash` as well?

Reply:
Yes, it should.